src/main/kotlin/cacophony/pipeline/CacophonyPipeline.kt

```diff
@@ -240,4 +242,34 @@ class CacophonyPipeline(
         asm.forEach { (function, asm) -> println("$function generates asm:\n$asm") }
         return asm
     }
+
+    fun saveAsm(asm: Collection<String>, dest: Path) {
+        dest.writeLines(asm, Charsets.UTF_8, StandardOpenOption.CREATE)
```

Comment:
Is this flag (StandardOpenOption.CREATE) necessary? I think this can be a simple dest.writeLines(asm). Also given the new signature this function should be called writeLines or something like that. At this point maybe just inline it in compile to not think about it?